.gitignore

```diff
@@ -13,3 +13,4 @@
 
 # original backup files
 *.orig
+/build/
```

Comment:
New line missing.

Reply:
not sure how this was coming in but this file should not be changed